<launch>
  <node name="environmentManagerGazebo" pkg="robot_ddpg_gazebo" type="robot_ddpg_gazebo_node"/>
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find robot_ddpg_gazebo)/worlds/slow.world"/>
    <arg name="gui" value="true"/>
  </include>
</launch>
